glotaran/model/model.py

```diff
@@ -253,6 +258,30 @@ def global_megacomplex(self) -> dict[str, Megacomplex]:
         """Alias for `glotaran.model.megacomplex`. Needed internally."""
         return self.megacomplex
 
+    def as_dict(self) -> dict:
+        model_dict = {}
+        model_dict["default-megacomplex"] = self.default_megacomplex
+
+        for name in self._model_items:
```

Comment:
```suggestion
        for item_name in self._model_items:
```
For consistencies sake (see line 278).

Reply:
done